index.html

```diff
@@ -81,148 +81,155 @@ <h4 class="list-heading">Skills</h4>
-            <ul class="programs">
-                <li class="lang">css &nbsp;|</li>
-                <li class="lang">html &nbsp;|</li>
-                <li class="lang">bootstrap |</li>
-                <li class="lang"> Ruby</li>
-            </ul>
-            <div>
-                <button class="button" type="button" onclick="">See Project</button>
+
+            <div class="work-section1">
+                <img src="images/snapshot.png" alt="Snapshoot">
+                <div class="divide1">
+                <h2>Multi-Post Stories </h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.</p>
+                <ul class="programs">
+                    <li class="lang">css &nbsp;|</li>
+                    <li class="lang">html &nbsp;|</li>
+                    <li class="lang">bootstrap |</li>
+                    <li class="lang"> Ruby</li>
+                </ul>
+                <div>
+                    <button class="button" type="button" onclick="">See Project</button>
+                </div>
+            </div>
             </div>
-        </div>
 
-        <div class="middle-section">
+            <div class="work-section">
+                <img src="images/snapshot.png" alt="Snapshoot">
+                <div class="divide">
+                <h2>Multi-Post Stories </h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.</p>
+                <ul class="programs">
+                    <li class="lang">css &nbsp;|</li>
+                    <li class="lang">html &nbsp;|</li>
+                    <li class="lang">bootstrap |</li>
+                    <li class="lang"> Ruby</li>
+                </ul>
+                <div>
+                    <button class="button" type="button" onclick="">See Project</button>
+                </div>
+                </div>
+            </div>
 
-            <hr>
-            <img src="images/snapshot.png" alt="Snapshoot">
-            <h2>Multi-Post Stories </h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
-                has been the industry's standard dummy text ever since the 1500s, when an unknown
-                printer took a standard dummy text.</p>
-            <ul class="programs">
-                <li class="lang">css &nbsp;|</li>
-                <li class="lang">html &nbsp;|</li>
-                <li class="lang">bootstrap |</li>
-                <li class="lang"> Ruby</li>
+            <div class="work-section1">
+                <img src="images/snapshot.png" alt="Snapshoot">
```

Comment:
- [ ] Please  🙏🏽 review the source of the images to show them in the browser. I could suggest using `./images/snapshot.png` instead of  `images/snapshot.png`.